public_dropin_gpu_environments/nemollm_inference_ms/start_server.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+# Copyright 2024 DataRobot, Inc. and its affiliates.
+#
+# All rights reserved.
+# This is proprietary source code of DataRobot, Inc. and its affiliates.
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+echo "Starting Custom Model environment with NIM"
+set -e
+
+GPU_COUNT=$(nvidia-smi -L | wc -l)
+echo "GPU count: $GPU_COUNT"
+
+if [ "$GPU_COUNT" -eq 0 ]; then
```

Comment:
How does shell treat empty string, does that map to zero? I'm thinking of the case where smi binary doesn't even exist in image so you will end up with an empty string.

Reply:
we counting lines
```
GPU_COUNT=$("" | wc -l)
echo "GPU count: $GPU_COUNT"
```
echoes
```
GPU count: 0
```

Reply:
> case where smi binary doesn't even exist

then startup script should exit, because of `set -e`.

Reply:
I didn't think set -e took effect for captured commands (I.e. $(command...))

But I did a quick test and you are right, wc outputs zero so we are all good.